internal/process/defunct_processes.go

```diff
@@ -0,0 +1,59 @@
+package process
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// DefunctProcesses returns the PIDs of all the zombie processes in the node
+func DefunctProcesses() ([]int, error) {
+	directories, err := os.Open("/proc")
+	if err != nil {
+		return nil, err
```

Comment:
Let's wrap errors to add more context:
```suggestion
		return nil, errors.Wrap(err, "read proc file system")
```

Reply:
1. `os.Open` provides enough context (as it returns `&os.PathError` with the file name, the op and the error) so I don't think it makes sense to wrap this.

2. Strictly speaking, we need to get rid of pkg/errors and use standard go 1.13+ error wrapping, i.e. `fmt.Errorf` with `%w`. But that's surely out of scope for this PR.